src/main/java/edu/cornell/kfs/sys/batch/PurgeConcurTablesStep.java

```diff
@@ -0,0 +1,27 @@
+package edu.cornell.kfs.sys.batch;
+
+import java.util.Date;
+
+import org.kuali.kfs.sys.batch.AbstractStep;
+
+import edu.cornell.kfs.concur.batch.service.ConcurTablesPurgeService;
+
+public class PurgeConcurTablesStep extends AbstractStep {
```

Comment:
Do we need a step specific for Concur or could we have a generic step and have the tables to be purged/or BOs configured in Spring?